packages/shared/models/user.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from 'mongoose'
+
+const userSchema = new mongoose.Schema({
+  id: String,
+  username: String,
+  passwordHash: String,
+  avatarURL: String,
```

Comment:
url? image content should be stored with type BSON 
https://docs.mongodb.com/manual/reference/bson-types/